code/modules/vending/vending.dm

```diff
@@ -1581,11 +1581,22 @@
 	slogan_list = list("A revolution in the pizza industry!",
 	"Prepared in moments!",
 	"I'm a chef who works 24 hours a day!")
+	var/sharpen = 0
 
 	lr = 1
 	lg = 0.6
 	lb = 0.2
 
+	attackby(obj/item/W, mob/user)
+		if (!sharpen && istype(W, /obj/item/kitchen/utensil/knife/pizza_cutter/traitor))
+			sharpen = 1
+			add_fingerprint(user)
+			boutput(user, "You jam the pizza sharpener inside the vending machine.")
```

Comment:
this whole mechanic is pretty strong, easily farmable pre-sharpened pizzas

Reply:
Sharpened pizzas are probably strongest with fun reagents in them, and the pizzas from the vending machine can't do that.

Reply:
can you not put reagents on the pizza afterwards, like all food items?

Reply:
I'm pretty sure syringes only allow you to put reagents into food items that don't have their own method of adding reagents.

Reply:
Syringes allow you to inject pizzas just fine. If you want to prevent that you'll need to override `is_open_container` to return `0` probably (though `is_open_container` is awful so even that might not work).

Reply:
In any case, if you try to get the pizzas from the sabotaged vending machine, they're getting launched at you